server/server.js

```diff
@@ -40,6 +42,17 @@ const setupProxy = (fraPath, tilTarget, headers = undefined) =>
         },
     });
 
+const manifestMedFlereFiler = (...filUrlStrenger) => {
```

Comment:
Trenger denne å være generell, egentlig? Kan det tenkes at vi skal utvide manifestet med flere ting enn env-fila?

Reply:
Hvis det kan gjøres litt enklere og eksplisitt så burde vi kanskje starte med det?
Så kan vi gjøre den generell om vi har behov for det senere...

Reply:
Grunnen til at jeg gjorde det slik var at det føltes mere deklarativt på denne måten. At vi sier hvilken fil som skal være med i server get.

Reply:
Har endret slik at bare en fil støttes